site/app/templates/autograding/AutoChecks.twig

```diff
@@ -64,6 +64,10 @@
                 <span onclick="openPopUp('{{ popup_css_file }}', '{{ element.title }}', {{ index }}, {{ check_index }}, {{ side }});" class="key_to_click" tabindex="0">
                     <i class="fas fa-window-restore" style="cursor: pointer;"></i>
                 </span>
+
+                <span onclick="downloadTestCaseResult({{ index }}, '{{ check.filename }}', {{ display_version }}, '{{ gradeable_id }}', '{{ who }}')" tabindex="0">
+                    <i class="fas fa-download" style="cursor: pointer;"></i>
+                </span>
```

Comment:
Although this button only downloads the student's output, it also appears on the expected program output diff element. It should just not show up at all for that element.